test/benchmark_http_client_test.cc

```diff
@@ -46,50 +77,56 @@ class BenchmarkClientHttpTest : public Test {
 
     auto& tracer = static_cast<Envoy::Tracing::MockHttpTracer&>(*http_tracer_);
     EXPECT_CALL(tracer, startSpan_(_, _, _, _))
-        .WillRepeatedly(
-            Invoke([&](const Envoy::Tracing::Config& config, const Envoy::Http::HeaderMap&,
-                       const Envoy::StreamInfo::StreamInfo&,
-                       const Envoy::Tracing::Decision) -> Envoy::Tracing::Span* {
-              EXPECT_EQ(Envoy::Tracing::OperationName::Egress, config.operationName());
-              auto* span = new NiceMock<Envoy::Tracing::MockSpan>();
-              return span;
-            }));
-    request_generator_ = []() {
-      auto header = std::make_shared<Envoy::Http::TestRequestHeaderMapImpl>(
-          std::initializer_list<std::pair<std::string, std::string>>(
-              {{":scheme", "http"}, {":method", "GET"}, {":path", "/"}, {":host", "localhost"}}));
-      return std::make_unique<RequestImpl>(header);
-    };
+        .WillRepeatedly([](const Envoy::Tracing::Config& config, const Envoy::Http::HeaderMap&,
+                           const Envoy::StreamInfo::StreamInfo&,
+                           const Envoy::Tracing::Decision) -> Envoy::Tracing::Span* {
+          EXPECT_EQ(Envoy::Tracing::OperationName::Egress, config.operationName());
+          auto* span = new NiceMock<Envoy::Tracing::MockSpan>();
+          return span;
+        });
   }
 
-  void testBasicFunctionality(const uint64_t max_pending, const uint64_t connection_limit,
-                              const uint64_t amount_of_request) {
+  // Primary testing method. Confirms that connection limits are met and number of requests are
+  // correct. If header expectations is not null, also checks the header expectations, if null, it
+  // is ignored.
+  void TestBenchmarkClientProcessesExpectedInflightRequests(
```

Comment:
The name of this function seems inconsistent with the rest of the style in this file. Most function names start with a lowercase letter. We likely cannot use a name starting with lowercase `test` since the testing framework might interpret this as a name of a test method. Can we choose a different name that will adhere to our style, e.g.:

`verifyBenchmarkClientProcessesExpectedInflightRequests`

Reply:
done.